ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -91,8 +95,14 @@ let prototyped_of_message = function
       Some "22.26.0"
   | "VTPM", "create" ->
       Some "22.26.0"
+  | "host", "apply_recommended_guidances" ->
+      Some "23.18.0"
```

Comment:
This addition doesn't seem right

Reply:
This surprised me as well, but I think it's ok... The datamodel has an explicit:

        ~lifecycle:[(Prototyped, "23.18.0", ""); (Removed, "23.25.0", "")]

I was perhaps manually removed from datamodel_lifecycle.ml and then added to datamodel_host.ml, but the automation put it back.